custom-scope-handler/src/main/java/org/wso2/custom/scope/handler/internal/CustomScopeHandlerServiceComponent.java

```diff
@@ -29,10 +29,12 @@
 
 @Component(name = "org.wso2.custom.scope.handler", immediate = true)
 public class CustomScopeHandlerServiceComponent {
```

Comment:
May need to change this similar to in[1], to keep consistency.

[1] https://github.com/wso2-incubator/samples-is/pull/18/files#r269278363

Reply:
This is a different component (custom scope handler).